sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -337,7 +337,7 @@ object DateTimeUtils {
 
   def microsToInstant(us: Long): Instant = {
     val secs = Math.floorDiv(us, MICROS_PER_SECOND)
-    val mos = Math.floorMod(us, MICROS_PER_SECOND)
+    val mos = us - secs * MICROS_PER_SECOND
```

Comment:
For better readability, could you leave some comments about the calculation reuse here?

Reply:
I added a comment